mediathread/templates/clientside/collectionwidget.mustache

```diff
@@ -116,6 +109,31 @@
     <div class="collection-assets annotation-embedding">
         <div id="asset_table" class="asset-table"></div>
     </div>
+
+    {{#paginator}}
```

Comment:
A slight nit -- maybe wrap the `paginator` block to only display if assets exist. You can use:
```
   {{#asset_count}}
      ...all the paginator code
   {{/#asset_count}}
```

Reply:
Got it!